Muhasin/index.js

```diff
@@ -0,0 +1,35 @@
+// Variables
+let apples = 20;
+let oranges = 30;
+let mangos = 5;
+
+// Are apples and oranges equal?
+console.log(apples == oranges); // == false
+
+// Are apples and oranges unequal ?
+console.log(apples == oranges); // == true
```

Comment:
2nd exercise- get rid of line 10. Your answer is wrong. 
The correct solution is :
console.log(apples !== oranges);